src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,76 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+
+  &__logo {
+    display: inherit;
+    align-self: center;
+  }
+
+  &__menu {
+    display: inherit;
+  }
+
+  &__list {
+    display: inherit;
+  }
+
+  &__item {
+    display: inherit;
```

Comment:
Do you really needs to inherit on all of this elements?

Reply:
`display: inherit` is used to simplify setting the properties for elements